app/views/bills/index.html.erb

```diff
@@ -32,5 +32,21 @@
       </select>
     </div>
   </div>
+  <div class="row">
+    <div class="col-6">
+      <h4>Five recent bills:</h4>
```

Comment:
This is really large on my screen. Can we update the size to match the "Enter a bill number to compare to your selected representative (Ex: 245-116)" and "Search bills by topic to compare" fonts?